<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QGridLayout" name="gameMiddleLayout">
     <item row="1" column="3">
      <widget class="QLCDNumber" name="turnsLeft"/>
     </item>
     <item row="1" column="0" rowspan="2">
      <layout class="QGridLayout" name="dominosRow1Layout"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>pakli mérete:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="deckSize"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>hátra lévő körök:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" rowspan="2">
      <layout class="QGridLayout" name="dominosRow2Layout"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>egygépes játék esetén a többiek táblája is meg kell jelenjen a főablakban</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>valahol itt lesz a kozos jatekter:2 sor dominó, és a pakli</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="playerLayout"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
